app/serializers/user_notification_total_serializer.rb

```diff
@@ -0,0 +1,58 @@
+# frozen_string_literal: true
+
+class UserNotificationTotalSerializer < ApplicationSerializer
+  attributes :id,
+             :username,
+             :unread_notifications,
+             :unread_personal_messages,
+             :unseen_reviewables,
+             :topic_tracking,
+             :group_inboxes
+
+  def unread_notifications
+    object.all_unread_notifications_count - object.new_personal_messages_notifications_count
+  end
+
+  def unread_personal_messages
+    object.new_personal_messages_notifications_count
+  end
+
+  def unseen_reviewables
```

Comment:
I think we can add an `include_unseen_reviewables?` to skip this attribute when user is not staff per what we are asserting for in the test.